coding/python_style_guide.rst

```diff
@@ -124,7 +121,7 @@ Flake8 command line invocation
 
 .. code-block:: bash
 
-   flake8 --ignore=E133,E226,E228,N802,N803 --max-line-length=110 .
```

Comment:
Reiterating: we're not going to merge this unless an acceptable automated method of shortening existing line lengths is implemented.  On the other hand, if an acceptable automated method of checking documentation line length separate from code line length is implemented, this will be adapted to match that.  In the meantime, with neither of those available, we continue with the existing policy.